server/tests/planning_export/planning_export_tests.py

```diff
@@ -526,7 +526,7 @@ def test_export_week(self):
                 ),
                 dutch_template_data,
             )
-            self.assertIn("<h3>zondag 21 april</h3>", dutch_template_data)
+            self.assertIn("<h3>Zondag 21 april</h3>", dutch_template_data)
```

Comment:
what about `April` that shouldn't be capitalized?

Reply:
not sure, as they only want the first letter of the heading to be capitalized.